modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -54,4 +54,12 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`spec.storage.managementState`
+
+|The Image Registry Operator sets the `spec.storage.managementState` parameter to `Managed` on new installations or upgrades of installer-provisioned infrastructure clusters on AWS or Azure.
+
+* `Managed`: Determines that the Image Registry Operator manages underlying storage. If the Image Registry Operator's `managementState` is set to `Removed`, then the storage is deleted.
+* `Unmanaged`: Determines that the Image Registry Operator leaves the storage settings alone. If the Image Registry Operator's `managementState` is set to `Removed`, then the storage is not deleted. If you provided an underlying storage unit configuration, such as a bucket or container name, the `spec.storage.managementState` parameter is set to `Unmanaged`.
+
```

Comment:
Just a small change here:

"If you provided an underlying storage unit configuration, such as a bucket or container name, **and the `spec.storage.managementState` is not yet set to any value**, the operator will configure it to `Unmanaged`"

Reply:
Updated